airbyte-integrations/connectors/source-twilio/source_twilio/schemas/accounts.json

```diff
@@ -4,11 +4,9 @@
       "type": ["null", "string"]
     },
     "date_created": {
-      "format": "date-time",
```

Comment:
Why are you removing the `date-time` format?

Reply:
One of the tests weren't passing because of it